Assets/Scripts/Gamestate/PlayerIdentity.cs.orig

```diff
@@ -4,9 +4,6 @@ using UnityEngine;
 
 public class PlayerIdentity : MonoBehaviour
 {
-<<<<<<< HEAD
-    private Color color;
-=======
```

Comment:
Nice that you removed the merge markers, though you should still **remove this file**.

Reply:
This goes for the .orig version of Menu as well. Git takes care of previous versions. You could try to checkout versions of a file from the last commit instead of creating a bunch of these superfluous files.